js/musicutils.js

```diff
@@ -43,7 +43,8 @@ const MUSICALMODES = {
 
      // 8 notes in an octave
     'ALGERIAN': [2, 1, 2, 1, 1, 1, 3, 1],
-    'DIMINISHED': [2, 1, 2, 1, 2, 1, 2, 1],
+    'DIMINISHED-WHOLE': [2, 1, 2, 1, 2, 1, 2, 1],
```

Comment:
I'd prefer to use spaces rather than hyphens but I suppose I could test for either. Eventually we need a pull-down menu for these (and/or a widget).